tools/deb/istio-iptables.sh

```diff
@@ -188,6 +188,7 @@ set -x # echo on
 # In both chains, '-j RETURN' bypasses Envoy and '-j ISTIO_REDIRECT'
 # redirects to Envoy.
 iptables -t nat -N ISTIO_REDIRECT
+iptables -t nat -A ISTIO_REDIRECT -p tcp --dport 53 -j RETURN
```

Comment:
why? To allow DNS over TCP?
If so, you have to also change the code in pilot to ignore port 53 from the list of ports that Pilot processes.

Reply:
@rshriram 

Many Thanks for this review.  Yes, this change is to not redirect DNS over TCP which is on port 53.  I will modify the codes in pilot as well, so please close this PR for now.

Reply:
I'm not entirely sure, long term (in particular when UDP support is added to envoy) we should intercept 53.
We discussed even handling 53 in the pilot agent.  Encrypting DNS and possibly redirecting it in case of multi-cluster is quite useful.

But short term - it is probably harmless, since we don't capture udp 53 we can also leave 53 free.